### ISSUE TITLE
ubuntu16.04LTS+pytorch0.4.1+cuda10.0+cuDNN

### ISSUE THREAD

**Issue by TsuiPaoPao** [xingyizhou/CenterNet#796]
Thanks for your good work.And I wanna ask which version of cuda and cuDNN you choose in your work?
If I run main.py in the conda env of the configuration as the title, then _**RuntimeError: cuda runtime error (11) : invalid argument at /pytorch/aten/src/THC/THCGeneral.cpp:663**_". I've searched the solution about this error and tried "**_pip3 install -U *torch1.0.0.whl_**", but it come out another error:"**_ImportError: torch.utils.ffi is deprecated. Please use cpp extensions instead_**". I've also searched the solution but it needs to install **_pytorch<=1.0_**. It seems like a crazy error loop.

**Comment by zxc916443179** [comment]
yeah, I have the same problem. My GPU device is RTX2080ti and after I've tried two days for installation I finally found that the cuda version author used is noncompatible on my environment. It's miserable

**Comment by TsuiPaoPao** [comment]
> yeah, I have the same problem. My GPU device is RTX2080ti and after I've tried two days for installation I finally found that the cuda version author used is noncompatible on my environment. It's miserable

I've also tried cuda9.0.  It failed. @…@

**Comment by mblak3** [comment]
Not sure if this will solve your problem, but installing pyTorch 0.4.1 with the following command: 
`conda install pytorch=0.4.1 cuda92 -c pytorch`

Specifying Cuda 9.2 allowed me to run the code, when previously I was stuck in the same error loop as you.
